addon/components/sl-modal.js

```diff
@@ -0,0 +1,236 @@
+import Ember from 'ember';
+import layout from '../templates/components/sl-modal';
+
+/**
+ * @module
+ */
+export default Ember.Component.extend({
+    // -----------------------------------------------------
+    // Dependencies
+
+
+    // -----------------------------------------------------
+    // Attributes
+
+
+    /** @type {String[]} */
+    attributeBindings: [
+        'aria-describedby',
+        'aria-hidden',
+        'aria-labelledby',
+        'tabindex'
+    ],
+
+    /** @type {String[]} */
+    classNames: [
+        'modal fade'
+    ],
+
+    /** @type {String} */
+    tagName: 'div',
+
+    /** @type {Object} */
+    layout: layout,
+
+
+    // -----------------------------------------------------
+    // Actions
+
+
+    /** @type {Object} */
+    actions: {
+        /**
+         * Trigger hiding the model
+         *
+         * @function actions:hide
+         * @returns {undefined}
+         */
+        hide() {
+            this.hide();
+        },
+
+         /**
+         * Trigger showing the model
+         *
+         * @function actions:show
+         * @returns {undefined}
+         */
+        show() {
+            this.show();
+        }
+    },
+
+
+    // -----------------------------------------------------
+    // Events
+
+
+    // -----------------------------------------------------
+    // Properties
+
+
+    /**
+     * aria-describedby attribute
+     *
+     * @type {?String}
+     * */
+    'aria-describedby': null,
```

Comment:
This property needs to be listed in the demo documentation page as a property than can be populated.  Refer to 
https://github.com/softlayer/sl-ember-components/blob/master/addon/mixins/docs/modal.md#aria-support for format of it